lib/dl_api_lib/dl_api_lib_tests/db/control_api/test_info.py

```diff
@@ -96,3 +97,24 @@ def test_get_connector_icon(self, client):
             assert "data" in icon
             assert icon["data"].get("standard") is not None and len(icon["data"]["standard"]) > 0, icon
             assert icon["data"].get("nav") is not None and len(icon["data"]["nav"]) > 0, icon
+
+    @pytest.mark.parametrize(
+        "conn_type_name",
+        [conn_type.name for conn_type in ConnectionType if conn_type != ConnectionType.unknown],
+    )
+    def test_get_connector_icon(self, client, conn_type_name):
+        icons_resp = client.get(f"/api/v1/info/connectors/icons/{conn_type_name}")
+        assert icons_resp.status_code == 200, icons_resp.json
+
+        resp_data = icons_resp.json
+        assert "icon" in resp_data
+        icon_data = resp_data["icon"]
+        assert "type" in icon_data
+        assert icon_data["type"] == "data"
+        assert "data" in icon_data
+        assert icon_data["data"].get("standard") is not None and len(icon_data["data"]["standard"]) > 0, icon_data
```

Comment:
`assert len(icon_data["data"].get("standard", [])) > 0, icon_data`
in `test_get_connector_icons` as well